### PR TITLE
Feat/pd 3081

### PR DESCRIPTION
**Pull request by arimieandreea**

### CODE REVIEW

**Review by PatriciaRomaniuc — APPROVED** [review]

File: src/components.d.ts [hunk 1]
```diff
@@ -149,7 +155,9 @@ export namespace Components {
     */
     'active': boolean;
   }
-  interface PieStimulusLayout {}
+  interface PieStimulusLayout {
+    'allowedResize': boolean;
```
Comment (andreeapescar) [inline src/components.d.ts:159]: make it optional

File: src/components.d.ts [hunk 2]
```diff
@@ -275,6 +283,7 @@ declare namespace LocalJSX {
     * Simulates a correct response for the item. This property will only have this effect if the `hosted` property is false and player is running client-side-only.
     */
     'addCorrectResponse'?: boolean;
+    'allowedResize'?: boolean;
```
Comment (andreeapescar) [inline src/components.d.ts:286]: add comment

File: src/components/pie-player/pie-player.tsx [hunk 3]
```diff
@@ -148,6 +148,8 @@ export class Player {
   @Prop({mutable: false, reflect: false})
   version: string = VERSION;
 
+  @Prop() allowedResize: boolean = false;
```
Comment (andreeapescar) [inline src/components/pie-player/pie-player.tsx:151]: add comment

File: src/components/pie-stimulus-layout/pie-stimulus-layout.tsx [hunk 4]
```diff
@@ -11,24 +11,81 @@ export class PieStimulusLayout {
       el.style.height = `calc(100vh - ${boundRect.top}px`;
     }
   }
+  private resizer: HTMLDivElement;
+
+  // shows us if resizing is activated or not
+  @Prop() allowedResize: boolean = false;
+
+  @State() isResizing = false;
```
Comment (andreeapescar) [inline src/components/pie-stimulus-layout/pie-stimulus-layout.tsx:19]: add comment (what are these used for)

File: src/components/pie-stimulus-layout/pie-stimulus-layout.tsx [hunk 5]
```diff
@@ -11,24 +11,81 @@ export class PieStimulusLayout {
...
+    this.isResizing = false;
+  }
+
+
   render() {
     return (
       <div id="pie-stimulus-container">
-        <div id="stimulus"><slot name='stimulus' /></div>
-        <div id="item"><slot name='item' /></div>
+        <div id="stimulus"
```
Comment (andreeapescar) [inline src/components/pie-stimulus-layout/pie-stimulus-layout.tsx:76]: if we'd define the style above and make it {} if allowedResize is false, would it work?

**Review by andreeapescar — CHANGES_REQUESTED** [review]

**Comment by andreeapescar** [comment]
I have few comments on this:
1. All your commits have "chore: ...". The commit message type "chore" is typically used to indicate routine or administrative tasks that do not introduce new features, fix bugs, or make significant changes. In your case, you added a feature.
2. Please create a specific demo page for this new feature.
3. Check the issues I mentioned in the video I've sent you:
a. Resize bar not visible - I would also add some padding/margin/any spacing you're thinking
b. Weird behaviour on resizing when content is larger

**Pull request merged**